mods/cnc/weapons/other.yaml

```diff
@@ -65,7 +65,7 @@ Chemspray:
 Napalm:
 	Inherits: ^FlameWeapon
 	ReloadDelay: 4
-	Range: 2c0
+	Range: 5c0
```

Comment:
I thought this was necessary to make the aircraft start dropping bombs early enough, but I admit I don't remember testing it.